stories/example/override.stories.tsx

```diff
@@ -0,0 +1,136 @@
+import { Meta } from '@storybook/react';
+import {
+  RoiContainer,
+  RoiList,
+  RoiProvider,
+  useRoiActions,
+  useRoiState,
+} from '../../src';
+import { Image } from '../utils/Image';
+import { Layout } from '../utils/Layout';
+import { CSSProperties } from 'react';
+import { CommittedRoi } from '../../src/types/Roi';
+
+export default {
+  title: 'Example/Override',
+} as Meta;
+
+interface CustomColorData {
+  backgroundColor: CSSProperties['backgroundColor'];
+  selectedBackgroundColor: CSSProperties['backgroundColor'];
+  textColor: CSSProperties['color'];
+}
+
+const initialRois: Array<CommittedRoi<CustomColorData>> = [
+  {
+    id: '0000-1111-2222-3333',
+    x: 0,
+    y: 0,
+    width: 0.2,
+    height: 0.2,
+    label: 'A',
+    data: {
+      textColor: 'white',
+      selectedBackgroundColor: 'darkgreen',
+      backgroundColor: 'green',
+    },
+  },
+  {
+    id: '1111-2222-3333-4444',
+    x: 0.3,
+    y: 0,
+    width: 0.2,
+    height: 0.2,
+    label: 'B',
+    data: {
+      textColor: 'white',
+      selectedBackgroundColor: 'darkgreen',
+      backgroundColor: 'green',
+    },
+  },
+  {
+    id: '2222-3333-4444-5555',
+    x: 0,
+    y: 0.3,
+    width: 0.3,
+    height: 0.3,
+    label: 'C',
+    data: {
+      textColor: 'white',
+      selectedBackgroundColor: 'darkgreen',
+      backgroundColor: 'green',
+    },
+  },
+];
+
+export function OverrideDefaultStyleWithClassname() {
+  return (
+    <Layout>
+      <RoiProvider initialRois={initialRois}>
+        <div style={{ display: 'flex', flexDirection: 'column', gap: 5 }}>
+          <Internal />
```

Comment:
This button does not work. You can remove it since it does not relate to this story.